test/with_dummyserver/test_proxy_poolmanager.py

```diff
@@ -67,21 +77,31 @@ def teardown_class(cls) -> None:
         super().teardown_class()
         shutil.rmtree(cls.certs_dir)
 
-    def test_basic_proxy(self) -> None:
+    def test_proxy_verified_http_http(self) -> None:
```

Comment:
This used to be the most basic test for the proxy functionality, it had to check that the urllib3's part just works.
Let's not make its name imply that the only purpose of it is checking the `proxy_is_verified`/`is_verified` values.

I can see how splitting the two tests into four ones can help discover individual issues, let's name the new tests as `test_http_proxy_http_destination` or something like that.

Reply:
Right, my fault, forgot to restore the original 2 tests.

Reply:
=> restored original tests
=> renamed new tests to style `test_is_verified_http_proxy_to_http_target`